src/amperity/gocd/secret/vault/plugin.clj

```diff
@@ -25,6 +25,8 @@
 
 ;; ## Plugin Initialization
 
+;; (vault/renew-token client)
```

Comment:
should probably be removed unless there's a TODO above it with something actionable.